spec/spec_helper.rb

```diff
@@ -1,8 +1,10 @@
 require 'simplecov'
 require 'coveralls'
-require './lib/iosgen'
+require 'thor'
 
-SimpleCov.start
-Coveralls.wear!
+Dir['./lib/**/*.rb'].each { |f| require f }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.